### PR TITLE
Adding 30 new currencies option.

### PR DESCRIPTION
**Pull request by Jonathansoufer**

### CODE REVIEW

File: src/scenes/Balance/AccountsCarousel/index.js [hunk 1]
```diff
@@ -21,7 +21,35 @@ const CURRENCIES = [
   'AUD',
```
Comment (dbuarque) [inline src/scenes/Balance/AccountsCarousel/index.js]: Hey @Jonathansoufer, did you test all currencies on the app?

Comment (dbuarque) [reply src/scenes/Balance/AccountsCarousel/index.js]: I'm missing tests here

Comment (dbuarque) [reply src/scenes/Balance/AccountsCarousel/index.js]: Please include KRW, RUB

**Comment by Jonathansoufer** [comment]
all currencies added.

**Pull request merged**